lib/govuk_publishing_components/app_helpers/table_helper.rb

```diff
@@ -1,3 +1,6 @@
+# This is part of the Public API of this gem
+# Any changes to the signature will require a major version change
```

Comment:
Thanks for this. I'd move this to be above the self.helper method - since that method is part of the public API and that is the place with the signature.

I'd also suggest that it's only non-backwards compatible changes to the signature.

Reply:
Ah yes, makes sense. I've updated the comment.